rclcpp/src/rclcpp/clock.cpp

```diff
@@ -138,9 +144,7 @@ Clock::create_jump_callback(
   }
 
   // Lookup the per-object mutex
-  g_clock_map_mutex.lock_shared();
-  std::mutex & clock_mutex = g_clock_mutex_map.at(this);
-  g_clock_map_mutex.unlock_shared();
+  std::mutex & clock_mutex = get_clock_mutex(this);
   {
     std::lock_guard<std::mutex> clock_guard(clock_mutex);
```

Comment:
Why not this:

```suggestion
    std::lock_guard<std::mutex> clock_guard(g_clock_mutex_map.at(this));
```

And remove the line above?

Reply:
Good call.  Done in 685bd50